scripts/benchmark_db/bench.rb

```diff
@@ -46,6 +46,13 @@ def forest_version
 def default_config
   toml_str = syscall('./target/release/forest-cli', 'config', 'dump')
 
+  # Comment chain.policy section
```

Comment:
Perhaps we can remove this section. It's not like we are ever modifying it anyway.

Reply:
Yes, that would work too. But this was supposed to be here for customising if we spawn a new testnet.

Reply:
Actually I think this section is not used by Forest, when loading the config file.